mm2src/coins/eth.rs

```diff
@@ -473,6 +474,32 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    let hash = match H256::from_str(&req.tx_hash[2..req.tx_hash.len()]) {
+        Ok(h) => h,
+        Err(_) => {
+            return MmError::err(RawTransactionError::InvalidHashError(req.tx_hash));
```

Comment:
Please replace return statements with `?` implementing `From<...>` trait for error(s) or using `map_to_mm`, example

https://github.com/KomodoPlatform/atomicDEX-API/blob/e98c61217451f38f54ae5d60f3d3b012ab2ac962/mm2src/coins/eth.rs#L563